scripts/manual_reviewer_smoke_test.py

```
@@ -0,0 +1,325 @@
+    """Start the lightweight review server."""
+    repo_root = Path(__file__).resolve().parents[1]
+    print("=" * 70)
+    print("Starting Lightweight Review Server")
+    print("=" * 70)
+
+    cmd = [
+        sys.executable,
+        "-m",
+        "monailabel.main",
+        "start_server",
+        f"--app",
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)